gpt_migrate/utils.py

```diff
@@ -203,4 +203,14 @@ def find_and_replace_file(filepath,find,replace):
         testfile_content = file.read()
     testfile_content = testfile_content.replace(find,replace)
     with open(filepath, 'w') as file:
-        file.write(testfile_content)
\ No newline at end of file
+        file.write(testfile_content)
+def read_file_in_chunks(file_path, chunk_size):
+    """
+    Generator to read a file in chunks of bytes
+    """
+    with open(file_path, 'r') as file:
+        while True:
+            data = file.read(chunk_size)
```

Comment:
I think we need to read it in chunks of lines instead, and return an array

Reply:
thanks William! Tested it with the benchmarks and they run as expected?

Reply:
Hey @0xpayne sorry about the delay! I just wrote a new one, this should be much better